gradebookng/api/src/main/java/org/sakaiproject/grading/api/GradingCategoryType.java

```diff
@@ -26,6 +26,7 @@
  */
 public enum GradingCategoryType {
 
+    IGNORE_THIS(0),
```

Comment:
Yeah enums not starting at 0 and hibernate are not friendly!

I would prefer this:
```suggestion
    UNKNOWN_CATEGORY(0),
```